Dev_Fund_ZIP_Masters_Of_The_Universe

```diff
@@ -0,0 +1,418 @@
+# Masters of The Universe? (MoU) Dev Fund ZIP
+
+    ZIP: Unassigned {numbers are assigned by ZIP editors}
+    Title: Masters of The Universe?
+    Owners: NoamChom (Zcash forum)
+    Credit ZIP: "Hybrid Deferred Dev Fund" by Jason McGee
+    <https://github.com/zcash/zips/pull/860>
+    Credits: Daira Emma Hopwood, Str4d, Kris Nuttycomb
+            Jason McGee <aquietinvestor@gmail.com>
+            Skylar Saveland <skylar@free2z.com>
+            Peacemonger <usezcash@proton.me>
+            Gguy <babygguy@gmail.com
+    Status: Draft
+    Category: Consensus | Process
+    Created: 2024-07-03
+    License: MIT
+    Pull-Request: <https://github.com/zcash/zips/pull/868>
+    Title Credits: <https://www.versobooks.com/products/1684-masters-of-the-universe>
+
+
+## Terminology
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
+document are to be interpreted as described in BCP 14 [#BCP14]_ when, and only
+when, they appear in all capitals.
+
+The terms "Mainnet" and "Testnet" in this document are to be interpreted as
+defined in the Zcash protocol specification [#protocol-networks]_.
+
+The term "full validator" in this document is to be interpreted as defined in
+the Zcash protocol specification [#protocol-blockchain]_.
+
+The term "network upgrade" in this document is to be interpreted as described
+in ZIP 200 6 and the Zcash Trademark Donation and License Agreement (4 or
+successor agreement).
+
+The terms "block subsidy" and "halving" in this document are to be interpreted
+as described in sections 3.10 and 7.8 of the Zcash Protocol Specification. 2
+
+"Zcash Foundation", also called "ZF", refers to the 501(c)(3) nonprofit
+corporation of that name.
+
+"Section 501(c)(3)" refers to that section of the U.S. Internal Revenue Code
+(Title 26 of the U.S. Code).
+
+"Community Advisory Panel" refers to the panel of community members assembled
+by the Zcash Foundation.
+
+The terms "Testnet" and "Mainnet" are to be interpreted as described in section
+3.12 of the Zcash Protocol Specification
+
+
+## Abstract
+
+This proposal describes a structure for the Zcash MoU Development
+Fund, to be enacted in Network Upgrade 6 and last for 4 years. This Dev Fund
+would consist of 25% of the block subsidies, split into 2 slices:
```

Comment:
Bug: the abstract says 25% for 4 years, but the specification says 20% for 1 year.

Reply:
Updated the spec for consistency 
25 total as 17 lockbox, 8 ZCG
4 year duration
Revisitation requirement, if the NDFM is complete earlier than 4 years